packages/pg-pool/test/idle-timeout-exit.js

```diff
@@ -11,7 +11,6 @@ if (module === require.main) {
   pool.query('SELECT NOW()', (err, res) => console.log('completed first'))
   pool.on('remove', () => {
     console.log('removed')
-    done()
```

Comment:
I haven’t really looked at this, but there’s probably more to do here than quietly deleting a line. Does none of this code run at all, then?

Reply:
My mistake! This is correct.